v03_pipeline/lib/tasks/dataproc/create_dataproc_cluster.py

```diff
@@ -35,6 +38,8 @@ def get_cluster_config(reference_genome: ReferenceGenome, run_id: str):
                     'REFERENCE_GENOME': reference_genome.value,
                     'PIPELINE_RUNNER_APP_VERSION': Env.PIPELINE_RUNNER_APP_VERSION,
                 },
+                'service_account': service_account_credentials.service_account_email,
```

Comment:
Note we will also call `get_service_account_credentials()` downstream when we actually make the API request.  This ensures we properly scope and credential the cluster at spinup time.